src/service/AnnounceService.ts

```diff
@@ -221,161 +229,155 @@ export class AnnounceService {
                 );
             };
 
-            if (multisigAccountInfo) {
-                logger.info(
-                    `The node's main account is a multig account with Address: ${
-                        multisigAccountInfo.minApproval
-                    } min approval. Cosigners are: ${multisigAccountInfo.cosignatoryAddresses
-                        .map((a) => a.plain())
-                        .join(
-                            ', ',
-                        )}. The tool will ask for the cosigners provide keys in order to announce the transactions. These private keys are not stored anywhere!`,
-                );
-                const cosigners = await this.promptAccounts(
+            const cosigners: Account[] = [];
+
+            if (operatingPublicAccount) {
+                // ask for operatingAccount private key
+                const operatingAccount = Account.createFromPrivateKey(
+                    await CommandUtils.resolvePrivateKey(
```

Comment:
Really nice the way you also accept mutlsig operator accounts. It could happen that both operator and main accounts are multisig, so the bonded would be accepting/asking cosignatures from both.

Only ask for the operator private key when the operator account is not multisig, move the prompt to the else of `if (multisigAccountInfo) {`